sycl/test/basic_tests/handler/handler_generic_integral_lambda.cpp

```diff
@@ -0,0 +1,74 @@
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//==---- handler_generic_integral_lambda.cpp - SYCL parallel_for generic and integral lambda test ----==//
```

Comment:
done

Reply:
I change the description in the header to fit 80 symbols. As we discussed just remove the comment and leave the file name since it's quite expressive.